docs/learn/dapp-developer/read-profile-data.md

```diff
@@ -7,115 +7,106 @@ description: Learn how to read profile data from your Universal Profile.
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# Read Universal Profile Data
+# Read Data from your Universal Profile
 
-<div style={{textAlign: 'center', color: 'grey'}}>
-  <img
-    src={require('/img/learn/up_view.png').default}
-    alt="Universal Profile example on universalprofile.cloud"
-    width="800"
-  />
-<br/>
-<i>A <a href="https://wallet.universalprofile.cloud/0x6979474Ecb890a8EFE37daB2b9b66b32127237f7?network=testnet">Universal Profile</a> as seen on UniversalProfile.cloud</i>
-<br /><br />
-</div>
-
-:::info
-
-The full code of this example can be found in the 👾 [lukso-playground](https://github.com/lukso-network/lukso-playground/tree/main/universal-profile) repository.
-
-:::
-
-:::tip Universal Profile Explorer
-
-The explorer on 🔮 [universalprofile.cloud](https://universalprofile.cloud/) indexes all deployed Universal Profiles on the LUKSO network. You can try out the following examples with any Universal Profile address.
-
-:::
-
-## Setup
-
-To easily interact with a profile you should use the ⚒️ [erc725.js](https://npmjs.com/package/@erc725/erc725.js) library. It is able to fetch and encode profile and contract data easily. You can install it in your project using:
+## Install [erc725.js](https://npmjs.com/package/@erc725/erc725.js) library
 
 ```shell
 npm install @erc725/erc725.js
 ```
 
-## Get all profile data keys
+## Create erc725 instance
 
-:::tip
+:::info How is data stored on a Universal Profile
 
-🔍 You can inspect a profile smart contract (or any ERC725 contract) using 🔎 [ERC725 Inspect](https://erc725-inspect.lukso.tech/inspector) to see all its 🗂️ [ERC725Y data keys](../../standards/lsp-background/erc725#erc725y-generic-data-keyvalue-store).
+Each data on a Universal Profile is stored into a generic key-value [ERC725Y](https://github.com/ethereum/ercs/blob/master/ERCS/erc-725.md#erc725y) store which allows storing an unlimited amount of data in the smart contract.
 
-:::
+ERC725Y standardizes a mapping of data keys to data values in order to have the ability to add or remove data across time without the need of redeploying the contract. It gives flexibility to the contract storage. On the smart contract, data keys are represented as bytes32 values and data values under these keys are stored as bytes.
 
-<details>
-<summary>
-<a href="../../standards/universal-profile/lsp3-profile-metadata">LSP3 - Profile Metadata</a> describes the data in the Universal Profile contract's <a href="../../standards/lsp-background/erc725#erc725y-generic-data-keyvalue-store">ERC725Y data storage</a>. You can get the content of these data keys directly using the ⚒️ <a href="../../tools/erc725js/classes/ERC725#getdata"> erc725.js</a>  library. 👇
-</summary>
-
-<div>
+LSP2 - ERC725YJSONSchema is the standard used by the LSPs to organize how the data should be represented as bytes32/bytes pairs.
```

Comment:
maybe you can link to:

https://docs.lukso.tech/standards/generic-standards/lsp2-json-schema

Reply:
```suggestion
[LSP2 - ERC725YJSONSchema](../../../[standards/generic-standards/lsp2-json-schema](https://docs.lukso.tech/standards/generic-standards/lsp2-json-schema)) is the standard used by the LSPs to organize how the data should be represented as bytes32/bytes pairs.
```